analyses/cell-type-ewings/references/msigdb-gene-sets.tsv

```diff
@@ -0,0 +1,11 @@
+name	geneset	category	subcategory
+staege	STAEGE_EWING_FAMILY_TUMOR	C2	CGP
+miyagawa_up	MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_UP	C2	CGP
+miyagawa_down	MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_DN	C2	CGP
+zhang	ZHANG_TARGETS_OF_EWSR1_FLI1_FUSION	C2	CGP
+riggi_up	RIGGI_EWING_SARCOMA_PROGENITOR_UP	C2	CGP
+riggi_down	RIGGI_EWING_SARCOMA_PROGENITOR_DN	C2	CGP
+kinsey_up	KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_UP	C2	CGP
+kinsey_down	KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_DN	C2	CGP
+hallmark_EMT	HALLMARK_EPITHELIAL_MESENCHYMAL_TRANSITION	H	
+gobp_ECM	REGULATION_OF_EXTRACELLULAR_MATRIX_ORGANIZATION	C5	BP

```

Comment:
I noticed that none of the result TSVs were associated with this gene set, and when I poked around more with the code, indeed it seems anything that gets returned with `C5` & `BP` gets filtered out because the gene set name is not actually present in MSigDB output. 

But, good news, it's just a typo! https://www.gsea-msigdb.org/gsea/msigdb/cards/GOBP_REGULATION_OF_EXTRACELLULAR_MATRIX_ORGANIZATION. I replaced this term in the TSV file then there are indeed results.

So, this should be replaced and the code should be re-run. This may also need to be updated in READMEs.

```suggestion
gobp_ECM	GOBP_REGULATION_OF_EXTRACELLULAR_MATRIX_ORGANIZATION	C5	BP
```